client/components/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import SummaryOverall from './summaryOverall';
+import GraphOverall from './graphOverall';
+import SummaryRecent from './summaryRecent';
+import GraphRecent from './graphRecent';
```

Comment:
Personal preference  - I would capitalize your file names for the components as well for symmetry.

Reply:
Not a bad idea - I'm used to camelCasing stuff, but in the case of components, this makes sense - I will go ahead and change that for my next commit.